group_project_v2/templates/html/user_label.html

```diff
@@ -0,0 +1 @@
+<span class="group-project-user-label">{{ full_name }}</span>
```

Comment:
@smarnach yes, basically it should be a link to somewhere, but it's not ready yet, so only full name remained. Other context parameters will be needed when they actually decide where those links need to point to.